app/frontend/src/components/Autocomplete.tsx

```diff
@@ -56,6 +58,19 @@ export default function Autocomplete<
           id={id}
           label={label}
           helperText={error || helperText}
+          InputProps={
+            endAdornment
+              ? {
+                  ...params.InputProps,
+                  endAdornment: (
+                    <>
+                      {params.InputProps.endAdornment}
+                      {endAdornment}
```

Comment:
Is one of these the cross for clearing the input field?

Reply:
yes, or whatever mui autocomplete wants to put there